AUTHORS.md

```diff
@@ -7,63 +7,71 @@ License URL: https://creativecommons.org/licenses/by/4.0/
+Music/Town 1.mid            | (c) 2010 Juan Manuel Nigretti        | https://soundcloud.com/goodgameaudio
+Music/Vehicle 1.mid         | (c) 2010 Juan Manuel Nigretti        | https://soundcloud.com/goodgameaudio
+Panorama/Cosmos1.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Dawn1.png          | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Dawn2.png          | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Dimension Rift.png | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Night Sky1.png     | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Night Sky2.png     | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Planet1.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Planet2.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Planet3.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Sky1.png           | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Sky2.png           | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Sunset1.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Panorama/Sunset2.png        | (c) 2015 Tom Lemmens [CC-BY], [OGA-BY] | https://github.com/lemtom
+Sound/Cancel1.wav           | (c) 2009 LokiF [CC0]                 | https://opengameart.org/content/gui-sound-effects
+Sound/Cursor1.wav           | (c) 2012 Kenney [CC0]                | https://opengameart.org/content/51-ui-sound-effects-buttons-switches-and-clicks
+Sound/Decision2.wav         | (c) 2016 celestialghost8 [CC0]       | http://opengameart.org/content/8bit-sfx
+Sound/Item2.wav             | (c) 2009 LokiF [CC0]                 | https://opengameart.org/content/gui-sound-effects
+System/Royal.png            | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System/System.png           | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System/SystemA.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System/SystemB.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System/SystemC.png          | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2a.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2b.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
+System2/System2c.png        | (c) 2011 andwhyisit [CC-BY], [OGA-BY]  |
 Title/Title1.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title2.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title3.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
 Title/Title4.png            | (c) 2010 Francisco de la Peña [CC0]  | https://easyrpg.org
+
+[CC0]: https://creativecommons.org/publicdomain/zero/1.0/deed.de
```

Comment:
`.en` would have been better.

Reply:
Automatic redirects :/
Good catch fixing this the next time I make a PR here.